hack/tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
please rename it to hack/test-dockerized.sh

Reply:
Done